bcda/cclf/cclf.go

```diff
@@ -286,54 +246,42 @@ func (importer CclfImporter) ImportCCLFDirectory(filePath string) (success, fail
 	}
 
 	if len(cclfMap) == 0 {
-		importer.Logger.Info("Failed to find any CCLF files in directory")
-		return 0, 0, skipped, nil
+		importer.Logger.Info("Failed to find any valid CCLF files in directory")
+		return 0, failure, skipped, err
```

Comment:
Returning failures here since there are a couple of different failure modes from LoadCclfFiles that don't explicitly return an error, i.e.

- Zip file with correct filename, but missing CCLF0 or CCLF8 within the zip file itself
- Unable to open zip file

It was kinda like this already but I realize maybe the sensible thing here is to actually return an error if failures > 0. Open to opinions!